### DISCUSSION TITLE
当页面存在多个vc-viewer组件时，卸载其中一个组件，其他组件都会报错

### DISCUSSION THREAD

**Discussion by ChauPha** [zouyaoji/vue-cesium#214] [Q&A]
当页面存在多个vc-viewer组件时，卸载其中一个组件，事件会报错
![2](https://user-images.githubusercontent.com/24471468/174739197-d08b7eac-3ae8-41fe-b7f0-25a9050b944c.jpg)

**Comment by zouyaoji** [comment]
多个vc-viewer的话要指定一下 `removeCesiumScript` 为 false.

**Reply by ChauPha** [reply]
好的 多谢
